### DISCUSSION TITLE
图库顶部空白区域是什么内容？

### DISCUSSION THREAD

**Discussion by Omoinemie** [liuzhihang/halo-theme-hao#684] [Q&A]
### 是什么版本出现了此问题？

1.4.8

### 在线站点地址

http://www.web.vomiku.com:7080/photos

### 发生了什么？

顶部上面有一块空白区域，没找到设置内容得地方，是做什么用的？
![image](https://github.com/liuzhihang/halo-theme-hao/assets/31991443/cb55249b-9ba1-4721-97f4-4e8240b18233)

**Comment by carolcoral** [comment]
`页面`中选择你创建的`图库页面`里面的设置，设置封面图即可

<img width="698" alt="image" src="https://github.com/liuzhihang/halo-theme-hao/assets/8952687/31a9f93a-8e3f-4954-89b4-d9a2d2202594">
